app/controllers/users_controller.rb

```diff
@@ -60,4 +61,9 @@ def get_name(user1, user2)
     user = [user1, user2].sort
     "private_#{user[0].id}_#{user[1].id}"
   end
+
+  def show_themself
+    @user = User.find(params[:id])
+    redirect_to posts_path if current_user.id == @user.id
+  end
```

Comment:
Consider using a scope for this or something

Reply:
okay, i just avoid make the query, deleting line 66 and comparing 
```
current.id == params[:id].to_i
```

Reply:
Ok